### ISSUE TITLE
Can't install modules (php-mysql) for 7.2.6-apache-stretch Debian version

### ISSUE THREAD

**Issue by arimourao** [docker-library/php#654]
I'm having a hard time to install this php module(php-mysql) to run an legacy application that depends on mysqli. When I run inside the container the command  apt-get install php-mysql I get a "unable to locate" response. I tried to tweak the sources.list but to no avail. Can someone please shed some light here? Also, it shouldn't be such a pain to add modules as needed.

**Comment by Minipada** [comment]
@arimourao I am having the same issue, I tried also with the fpm image. How dd you solve it?

**Comment by arimourao** [comment]
I didn't. If you look at the [docs](https://hub.docker.com/_/php/) they compressed the PHP's source and it is too much hassle to install any extensions. I just moved on, I found an image in docker hub that suited my needs.

**Comment by tianon** [comment]
For `mysqli`, it should be as trivial as the following:

```dockerfile
FROM php:7.2-apache-stretch
RUN docker-php-ext-install mysqli
```

**Comment by Minipada** [comment]
It is not for mysql though

**Comment by tianon** [comment]
The `mysql` extension is only available on PHP 5 -- see http://us2.php.net/manual/en/mysql.php ("PHP 7 removes the mysql extension, leaving only the latter two options.").

**Comment by dwclogic** [comment]
Dockerfile used :: 
FROM php:apache
MAINTAINER David Caldwell <dave@dwclogic.com>
COPY ./httpdocs/php.ini /usr/local/etc/php/php.ini
RUN docker-php-ext-install mysqli
RUN docker-php-ext-enable mysqli
RUN apachectl restart
 
RESULT:
website    | PHP Warning:  PHP Startup: Unable to load dynamic library 'mysqli' (tried: /usr/local/lib/php/extensions/no-debug-non-zts-20180731/mysqli (/usr/local/lib/php/extensions/no-debug-non-zts-20180731/mysqli: cannot open shared object file: No such file or directory), /usr/local/lib/php/extensions/no-debug-non-zts-20180731/mysqli.so (/usr/local/lib/php/extensions/no-debug-non-zts-20180731/mysqli.so: cannot open shared object file: No such file or directory)) in Unknown on line 0

**This should have worked,** the files for mysqli are missing from the image entirely.  I started up this image, and searched the entire filesystem for the mysqli.so file, Nada.

I found the TEMPLATE file I think was where the image was built from and I think these lines are missing from it.  Dockerfile-debian.template [https://github.com/docker-library/php/blob/master/Dockerfile-debian.template](url)

		--enable-mysqli \
		--with-mysqli=mysqlnd \

I tried to build it, and there were many errors, I didn't have/find the needed configuration variables data.  I tried other images, but seriously someone has to just try images until one works????

Docker Hub needs to improve its Tag listings for Repositories, make them searchable, and or add Filtering so someone can whittle down all the options a bit.  PHP Repository literally has hundreds of tags.  Wasn't looking for anything special but Vanilla PHP 7 or 5.6 with mysqli that can be activate.
